CHANGELOG.rst

```diff
@@ -5,20 +5,34 @@ amazon.aws Release Notes
 .. contents:: Topics
 
 
+v5.5.3
+======
+
+Release Summary
+---------------
+
+This release contains a few bugfixes.
+
+Bugfixes
+--------
+
+- rds_cluster - Add ``AllocatedStorage``, ``DBClusterInstanceClass``, ``StorageType``, ``Iops``, and ``EngineMode`` to the list of parameters that can be passed when creating or modifying a Multi-AZ RDS cluster (https://github.com/ansible-collections/amazon.aws/pull/1657).
+- rds_cluster - Allow to pass GlobalClusterIdentifier to rds cluster on creation (https://github.com/ansible-collections/amazon.aws/pull/1663)."
+
 v5.5.2
 ======
 
 Release Summary
 ---------------
 
-This release brings few bugfixes.
+This release brings a few bugfixes.
 
 Bugfixes
 --------
 
 - cloudwatchevent_rule - Fixes changed status to report False when no change has been made. The module had incorrectly always reported a change. (https://github.com/ansible-collections/amazon.aws/pull/1589)
-- lambda_execute - Fixes to the stack trace output, where it does not contain spaces between each character. The module had incorrectly always outputted extra spaces between each character. (https://github.com/ansible-collections/amazon.aws/pull/1615)
```

Comment:
Why these changes?

Reply:
That happened automatically when I ran antsibull-changelog, I assumed it was putting the bugfixes in alphabetical order?

Reply:
@felixfontein Any idea why antsibull-changelog would change the order of an old changelog entry?

Reply:
Oh, I see something weird happened with the release note for the previous release in changelog.yaml, so it considered 5.5.2 fair game for reformatting.